resources/js/smoothScroll.js

```diff
@@ -27,15 +28,17 @@ function checkScrollPosition() {
 }
 
 // Show/hide the button based on scroll position
-checkScrollPosition();
 window.addEventListener("scroll", checkScrollPosition);
 
-smoothScroll(backToTopArrow, document.getElementById("home"));
-smoothScroll(
-    document.getElementById("scroll-to-contact-btn"),
-    document.getElementById("contact")
-);
-smoothScroll(
-    document.getElementById("scroll-to-projects-btn"),
-    document.getElementById("projects")
-);
+function ifElementExists(selectorButton, selectorSection) {
```

Comment:
A wording suggestion that is (IMO) more explicit about the goals
```suggestion
function registerWhenElementExists(selectorButton, selectorSection) {
```